test/helpers/cons.go

```diff
@@ -75,16 +75,20 @@ const (
 	HttpdImage = "cilium/demo-httpd"
 
 	// Names of commonly used containers in tests.
-
-	Httpd1 = "httpd1"
-	Httpd2 = "httpd2"
-	Httpd3 = "httpd3"
-	App1   = "app1"
-	App2   = "app2"
-	App3   = "app3"
-	Client = "client"
-	Server = "server"
-	Host   = "host"
+	Httpd1    = "httpd1"
+	Httpd2    = "httpd2"
+	Httpd3    = "httpd3"
+	App1      = "app1"
+	App2      = "app2"
+	App3      = "app3"
+	Client    = "client"
+	Server    = "server"
+	Host      = "host"
+	KafkaApp  = "kafka"
+	Zookeeper = "zook"
+	Backup    = "empire-backup"
+	EmpireHq  = "empire-hq"
+	Outpost   = "empire-outpost"
```

Comment:
why only outpost and not outpost-8888 and outpost-9999 as in the GSG?

Reply:
I would move this Backup, EmpireHq and Outpost variables to test. It's only used by the test and I don't think a test case that can be used too.

Reply:
It doesn't make a difference because the policies applied to both outpost-8888 and outpost -9999 is same. They merely behave as two separate visual terminals for the GSG are exactly the same in behavior.

Reply:
@eloycoto We have variables specific to HTTP tests defined here too. I can move it, but when we decide to add more Kafka tests, we might end up moving it again.

Reply:
If we add more kafka test it'll be added on `test/k8sT/KafkaPolicies.go` so no reason to have it on cons. 

The HTTP cons are used in `helpers/wrappers.go` and multiple runtime and k8st tests

Reply:
Yes. My point is just like HTTP, Kafka needs to have more test coverage, and will have it eventually.  We will end up using these more as the Kafka tests are extended. Hence I think this is a good generic place for it.  Anyways, this is a trivial issue to debate over. I will change it.